docs/UserGuide.md

```diff
@@ -249,6 +264,124 @@ __________________________________________
 ```
 <hr/>
 
+### Update the quantity of an item in Fridget: `update`
+Use this command to update the quantity of items in Fridget.
+
+Format: `update ITEM_NAME`
+
+<div style="display: inline-block;background-image: linear-gradient(180deg, #fff5d5, #fff3cd);  padding: 1rem; margin: 1rem; margin-left: 0; border-radius: 1em">
+:exclamation:
+The <b>ITEM_NAME</b> can be in a <ins>natural language</ins> format. 
+</div>
+
+<div style="display: inline-block;background-image: linear-gradient(180deg, #fff5d5, #fff3cd); padding: 1rem; margin: 1rem; margin-left: 0; border-radius: 1em; word-wrap: break-word">
+:exclamation:
+The <b>ITEM_NAME</b> should be the <ins>same</ins> as the name of the item you are trying to remove. 
+</div>
+
+<div style="display: inline-block;background-image: linear-gradient(180deg, #fff5d5, #fff3cd); padding: 1rem; margin: 1rem; margin-left: 0; border-radius: 1em; word-wrap: break-word">
+:exclamation:
+The <b>ITEM_NAME</b> can also be a <ins>part of the name</ins> of the item. 
+</div>
+
+**Example of usage:**
+
+<ins>If there is only one item with a name exactly the same as `ITEM_NAME`</ins>:
+```markdown
+USER INPUT: update burger
+__________________________________________
+How many of burger do you have left?
+__________________________________________
+USER INPUT: 5
+__________________________________________
+Quantity of burger is now 5.
+__________________________________________
+```
+
+<ins>If there is only one item with a name containing or with different upper or lower case to `ITEM_NAME`</ins>:
+```markdown
+USER INPUT: update Burger
+__________________________________________
+Did you mean: burger? [Y/N]
+__________________________________________
+USER INPUT: Y
+__________________________________________
+How many of burger do you have left?
+__________________________________________
+USER INPUT: 8
+__________________________________________
+Quantity of burger is now 8.
+__________________________________________
+```
+
+<ins>If there are multiple items with a name containing `ITEM_NAME`</ins>:
+```markdown
+USER INPUT: update burger
+__________________________________________
+Which item would you like to overwrite quantity? Type the index of the item below.
+1. cheese burger | Qty: 8 | 11 Nov 2022
+2. fish burger | Qty: 80 | 12 Dec 2022
+If you've changed your mind, simply type 'quit'.
+__________________________________________
+USER INPUT: 2
+__________________________________________
+How many of fish burger do you have left?
+__________________________________________
+USER INPUT: 32
+__________________________________________
+Quantity of burger is now 32.
+__________________________________________
+```
+
+<ins>If there are multiple quantities of the same item</ins>:
```

Comment:
Could this be changed to "If there are multiple items of the same name?"

Reply:
deon